guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_parent.adoc

```diff
@@ -2,8 +2,14 @@
 = Upgrading {ProjectServer}
 
 This section describes how to upgrade {ProjectServer} from {ProjectVersionPrevious} to {ProjectVersion}.
-You can upgrade from any minor version of {ProjectName} Server {ProjectVersionPrevious}.
+
 ifdef::satellite[]
+Update your {ProjectServer} {ProjectVersionPrevious} to the latest {ProjectVersionPrevious}.z release before proceeding.
+You can use the following command to update to the latest z-stream.
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+# satellite-maintain upgrade run --target-version {ProjectVersionPrevious}.z
```

Comment:
Even though this is for satellite-only, I'd recommend against using downstream-only attributes such as "satellite-maintain" in favor of `{foreman-maintain}`.